spec/requests/admin/users_controller_spec.rb

```diff
@@ -814,6 +814,96 @@
       end
     end
 
+    it "blocks the e-mail if block_email == true" do
+      user_emails = delete_me.user_emails.pluck(:email)
+
+      delete "/admin/users/#{delete_me.id}.json", params: { block_email: true }
+      expect(response.status).to eq(200)
+
+      expect(user_emails.size).to be >= 1
```

Comment:
While reading this assertion, one thing that isn't clear to me is what purpose does this assertion serve because we're asserting for a range instead of a definitive value. Does this mean the user email size changes between test runs?

Reply:
No I was testing if there is at least one e-mail to be blocked.

Reply:
ic. Personally, I would write the spec this way to make it clear what changes we're expecting as we execute the request. 

```suggestion
      expect do 
        delete "/admin/users/#{delete_me.id}.json", params: { block_email: true }
        expect(response.status).to eq(200)
      end.to change { ScreenedEmailed.where(email: delete_me.user_emails.pluck(:email)).count }.from(0).to(1)
```